src/sentry/static/sentry/app/components/broadcastModal.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import IconCloseLg from '../icons/icon-close-lg';
+import ConfigStore from '../stores/configStore.jsx';
+import ApiMixin from '../mixins/apiMixin';
+
+const ReleaseAnnouncement = ({close}) => {
+  const mediaUrl = ConfigStore.get('mediaUrl');
+  return (
+    <div>
+      <h3>
+        Releases are better with commits
+        {' '}
+        <span className="badge badge-square badge-new">NEW</span>
+      </h3>
+      <p>
+        <img src={mediaUrl + 'dist/release-commits-modal.gif'} />
+      </p>
+      <p>
+        <h5 style={{lineHeight: '1.2'}}>
+          By integrating commit data with Sentry, you’ll unlock a number of helpful features:
+        </h5>
+        <ul>
+          <li>
+            Enhanced releases overview page that allows you to see new and resolved issues, files changed and authors all in the same place
+          </li>
+          <li>
+            Resolving Sentry issues via commit messages
+          </li>
+          <li>
+            Suggested assignees for issues
+          </li>
+          <li>
+            Detailed summary emails when a deploy goes out
+          </li>
+        </ul>
+      </p>
+      <p className="release-buttons">
+        <a className="btn btn-default btn-lg" href="#" onClick={close}>
+          Dismiss
+        </a>
+        <a
+          className="btn btn-primary btn-lg"
+          href="https://blog.sentry.io/2017/05/01/release-commits.html"
+          onClick={close}>
+          Read the full post
+        </a>
+      </p>
+    </div>
+  );
+};
+
+ReleaseAnnouncement.propTypes = {
+  close: React.PropTypes.func.isRequired
+};
+
+const BroadcastModal = React.createClass({
+  propTypes: {
+    closeBroadcast: React.PropTypes.func.isRequired
+  },
+  mixins: [ApiMixin],
+
+  getInitialState() {
+    return {
+      alerts: [ReleaseAnnouncement],
+      index: 0
+    };
+  },
+
+  componentWillMount() {
+    document.addEventListener('keydown', this.handleKeyDown);
+  },
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.handleKeyDown);
+  },
+
+  close() {
+    //tell server to close
+    let user = ConfigStore.get('user');
+    let markedData = {options: {}};
+    markedData.options.seenReleaseBroadcast = true;
+
+    this.api.request(`/users/${user.id}/`, {
+      method: 'PUT',
+      data: markedData,
+      success: (data, _, jqXHR) => {
+        ConfigStore.set('user', data);
+        this.props.closeBroadcast();
+      },
+      error: err => {
+        this.props.closeBroadcast();
```

Comment:
wondering if there's anything we should do here to indicate to the user we had trouble saving their settings. like maybe we should just have one of those error indicators that says something like "Error saving dismissal" or "Error saving your preferences" .. or just something so that if we wind up showing it to them again, they'll at least know that there was some hiccup the first time they dismissed? not sure if other people have suggestions...